forge-gui/res/cardsfolder/upcoming/ichor_aberration.txt

```diff
@@ -0,0 +1,17 @@
+Name:Ichor Aberration
+ManaCost:1 U B
+Types:Creature Phyrexian Horror
+PT:3/3
+K:Flying
+K:Defender
+S:Mode$ CanAttackDefender | ValidCard$ Card.Self+powerGE7 | Description$ As long as CARDNAME's power is 7 or greater, it can attack as though it didn't have defender. 
+T:Mode$ Proliferate | ValidPlayer$ You | TriggerZones$ Battlefield | TriggerZones$ Battlefield | IsPresent$ Card.Self | PresentZone$ Battlefield | Execute$ TrigPerpetualBuff | TriggerDescription$ Whenever you proliferate, if CARDNAME is in your graveyard or on the battlefield, CARDNAME perpetually gets +1/+1. You may cast it from your graveyard this turn.
```

Comment:
One trigger should be better even if you need SVar then

Reply:
Hmn, I'll make an issue for later as it might be more involved than I initially thought

Reply:
Feel free to use the previous version, it's not a high priority